helm/policies-common/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+name: policies-common
```

Comment:
Do we need to prefix everything with `policies`? Especially for folders it seems redundant as the repo is already called `clusterpolicies`

Reply:
In places where the name is actually used I think it will make more sense. Like with opsctl it would currently be `opsctl deploy -i <whatever> policies-kvm`. I tried to keep this similar to the `releases` structure

Reply:
I understand that - but the folders in `releases` also don't have a prefix. Maybe I am missing something.

Reply:
The folders for the actual helm charts do https://github.com/giantswarm/releases/tree/master/helm

The outer folders are `kustomized` in, which we can't do with `ClusterPolicy` CRs because it alphabetizes things and re-quotes strings which break kyverno